app/src/main/java/com/x8bit/bitwarden/data/platform/util/CipherViewExtensions.kt

```diff
@@ -45,6 +45,8 @@ val CipherView.subtitle: String?
                     }
                 }
         }
+
+        CipherType.SSH_KEY -> null
```

Comment:
Will we need something here eventually

Reply:
`SSH_KEY` will not have a subtitle. Similar to `SECURE_NOTE`. I'll clean this up so they're grouped together.